cloudbuild.yaml

```diff
@@ -8,6 +8,17 @@ steps:
   - --tag=gcr.io/$PROJECT_ID/rcran-build:$BUILD_ID
   - .
 
+# Delete old images
+- id: 'delete-old-images'
+  name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    set -e
+    gcloud container images list-tags gcr.io/$PROJECT_ID/rcran-build --filter="NOT tags:latest timestamp.datetime < -P3M" --format='get(digest)' --limit 100 | xargs -I {} gcloud container images delete gcr.io/$PROJECT_ID/rcran-build@{} --quiet --force-delete-tags
+    gcloud container images list-tags gcr.io/kaggle-images/rcran --filter="NOT tags:latest timestamp.datetime < -P3M" --format='get(digest)' --limit 100 | xargs -I {} gcloud container images delete gcr.io/kaggle-images/rcran@{} --quiet --force-delete-tags
```

Comment:
Should we add the GPU images too?

Reply:
Why 3 months instead of 9 in the other projects?

Reply:
The `rcran` image doesn't have a GPU version. The GPU specific packages are installed in `rstats`.

Why 3 months, somewhat arbitrary. I figured we rarely debug the `rcran` image directly. I could be wrong. I can set it to 9 months. Or something else. WDYT?

Reply:
I will use 6 months everywhere.